readme.txt

```diff
@@ -6,27 +6,29 @@ Tested up to: 5.5.1
 Stable tag: 3.7.0
 License: BSD-3-Clause
 
-All of Cloudflare’s performance and security benefits in a simple one-click install of recommended settings specifically developed for WordPress.
+All of Cloudflare’s performance and security benefits in a simple one-click install.
 
 == Description ==
 
 = What this plugin can do for you =
 
+**Automatic Platform Optimization (APO)**
+
+Take your WordPress site's performance to the next level with Automatic Platform Optimizations (APO). APO allows Cloudflare to serve your entire WordPress site from its edge network ensuring consistent, fast performance for visitors no matter where they are. Find out more in our [announcement blog](https://blog.cloudflare.com/automatic-platform-optimizations-starting-with-wordpress/).
```

Comment:
The URL listed here is a 404.

Reply:
This is intended until the announcement is made